src/cli/reinstall.rs

```diff
@@ -41,6 +41,11 @@ pub struct Args {
     /// Install all environments.
     #[arg(long, short, conflicts_with = "environment")]
     pub all: bool,
+
+    /// Skip installation of specific packages present in the lockfile. Requires --frozen
+    /// This can be useful in particular for skipping local source dependencies.
+    #[arg(long, requires = "frozen")]
+    pub skip: Option<Vec<String>>,
```

Comment:
Not 100% how `--skip` should / will interact with `--packages` here. any thought?